test/mocks/electron.js

```diff
@@ -1,10 +1,23 @@
 'use strict';
+const sinon = require('sinon');
+const tempy = require('tempy');
+const path = require('path');
 
+const temporaryDir = tempy.directory();
+
+process.env.TZ = 'America/New York';
```

Comment:
This time zone identifier is invalid.

```suggestion
process.env.TZ = 'America/New_York';
```